.pre-commit-config.yaml

```diff
@@ -1,16 +1,19 @@
+# mind that a similar file exists for linting usage
 repos:
--   repo: https://github.com/ambv/black
-    # TODO switch to 'stable' once --skip-string-normalization lands on it
-    rev: master
+  - repo: https://github.com/ambv/black
+    rev: stable
     hooks:
-    - id: black
-      args: [--quiet, --safe, --skip-string-normalization]
-      python_version: python3.6
--   repo: https://github.com/pre-commit/pre-commit-hooks
+      - id: black
+        args: [--quiet, --safe, --skip-string-normalization, cerberus, setup.py]
+        python_version: python3.6
+        types:
+          - python
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v1.3.0
     hooks:
-    -   id: trailing-whitespace
-    -   id: end-of-file-fixer
```

Comment:
What is the reasoning behind removing `trailing-whitespace` and `end-of-file-fixer` hooks? Is Black going to take care of that (I admit I did not check the docs)

Reply:
yes, i checked both practically. otherwise black wouldn't be fun anyway.

Reply:
👍